spacy/tests/doc/test_span.py

```diff
@@ -98,6 +98,11 @@ def test_spans_span_sent(doc, doc_not_parsed):
     assert doc[:2].sent.root.text == "is"
     assert doc[:2].sent.text == "This is a sentence."
     assert doc[6:7].sent.root.left_edge.text == "This"
+    assert doc[0: len(doc)].sent == list(doc.sents)[0]
```

Comment:
I'd add an additional line to clarify the difference between `span.sent` and `span.sents`:
```suggestion
    assert doc[0 : len(doc)].sent == list(doc.sents)[0]
    assert list(doc[0 : len(doc)].sents) == list(doc.sents)
```